From bdcbeab0d8ac0963c61ae1d9d1ed57d3effb8ec3 Mon Sep 17 00:00:00 2001
From: FRIGN <dev@frign.de>
Date: Sun, 14 Feb 2016 01:32:02 +0100
Subject: [PATCH 2/7] Clarify config.def.h

Clear up the wording a bit and explain what failonclear means.
---
 config.def.h | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index fca0ae0..eae2d9a 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,6 +1,8 @@
 static const char *colorname[NUMCOLS] = {
 	"black",     /* after initialization */
 	"#005577",   /* during input */
-	"#CC3333",   /* failed/cleared the input */
+	"#CC3333",   /* wrong password */
 };
-static const Bool failonclear = True;
+
+/* treat a cleared input like a wrong password */
+static const int failonclear = 1;
-- 
2.9.2

